app/controllers/api/v0/videos_controller.rb

```diff
@@ -1,21 +1,23 @@
 module Api
   module V0
     class VideosController < ApiController
-      caches_action :index,
-                    cache_path: proc { |c| c.params.permit! },
-                    expires_in: 10.minutes
       respond_to :json
 
       before_action :cors_preflight_check
       after_action :cors_set_access_control_headers
 
       def index
```

Comment:
does this need a before_action for the cache control headers?
```ruby
before_action :set_cache_control_headers, only: %i[index]
```

Reply:
great catch!